synapse/app/__init__.py

```diff
@@ -24,7 +24,9 @@
 try:
     python_dependencies.check_requirements()
 except python_dependencies.DependencyException as e:
-    sys.stderr.writelines(e.message)
+    sys.stderr.writelines(
+        e.message  # noqa: B306, DependencyException.message is a property
```

Comment:
Should we just be using `str(e)` in each of these spots as the [bugbear docs](https://github.com/PyCQA/flake8-bugbear#python-3-compatibility-warnings) suggest?

Reply:
no, because `DependencyException.message` is a property that returns a custom value, if we'd wanna do that, we need to override `__str__` on `DependencyException`

Reply:
Hmmmm, thanks for that mention, ive subbed to it.

But in the meantime, would this be accepted? (`noqa` on this, with the reasoning behind it as well)